<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" >
 <class>preferencesDialog</class>
 <widget class="QDialog" name="preferencesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Preferences</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../ElmerGUI.qrc" >
    <normaloff>:/icons/Mesh3D.png</normaloff>:/icons/Mesh3D.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="meshPointsGroup" >
     <property name="title" >
      <string>Mesh points</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QRadioButton" name="meshPointsSurface" >
          <property name="text" >
           <string>Use surface mesh</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="sizeLabel" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Size:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QSpinBox" name="pointSize" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QRadioButton" name="meshPointsVolume" >
          <property name="text" >
           <string>Use volume mesh</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="qualityLabel" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Quality:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QSpinBox" name="pointQuality" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="minimum" >
           <number>2</number>
          </property>
          <property name="value" >
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QCheckBox" name="meshPointsClip" >
          <property name="text" >
           <string>Use clip plane</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="meshEdgesGroup" >
     <property name="title" >
      <string>Mesh edges</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QRadioButton" name="meshEdgesSurface" >
          <property name="text" >
           <string>Use surface mesh</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QRadioButton" name="meshEdgesVolume" >
          <property name="text" >
           <string>Use volume mesh</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="meshLineWidthLabel" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Line width:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QSpinBox" name="meshLineWidth" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>70</width>
            <height>24</height>
           </size>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QCheckBox" name="meshEdgeTubes" >
          <property name="text" >
           <string>Use tube filter</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QLabel" name="meshEdgeTubeQualityLabel" >
          <property name="text" >
           <string>Tube quality:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" >
         <widget class="QSpinBox" name="meshEdgeTubeQuality" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="minimum" >
           <number>3</number>
          </property>
          <property name="value" >
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2" >
         <widget class="QLabel" name="meshEdgeTubeRadiusLabel" >
          <property name="text" >
           <string>Tube radius:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3" >
         <widget class="QSpinBox" name="meshEdgeTubeRadius" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QCheckBox" name="meshEdgesClip" >
          <property name="text" >
           <string>Use clip plane</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="featureGroup" >
     <property name="title" >
      <string>Feature edges</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QRadioButton" name="surfaceRButton" >
          <property name="text" >
           <string>Use surface mesh</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="angleLabel" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Feature angle:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QSpinBox" name="angleSpin" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>70</width>
            <height>24</height>
           </size>
          </property>
          <property name="minimum" >
           <number>0</number>
          </property>
          <property name="maximum" >
           <number>90</number>
          </property>
          <property name="value" >
           <number>45</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QRadioButton" name="volumeRButton" >
          <property name="text" >
           <string>Use volume mesh</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="lineWidthLabel_2" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Line width:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QSpinBox" name="lineWidthSpin" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>70</width>
            <height>24</height>
           </size>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QCheckBox" name="featureEdgeTubes" >
          <property name="text" >
           <string>Use tube filter</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QLabel" name="tubeQualityLabel" >
          <property name="text" >
           <string>Tube quality:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" >
         <widget class="QSpinBox" name="featureEdgeTubeQuality" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="minimum" >
           <number>3</number>
          </property>
          <property name="value" >
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QCheckBox" name="featureEdgesClip" >
          <property name="text" >
           <string>Use clip plane</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" >
         <widget class="QLabel" name="tubeRadiusLabel" >
          <property name="text" >
           <string>Tube radius:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3" >
         <widget class="QSpinBox" name="featureEdgeTubeRadius" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QCheckBox" name="drawBoundaryEdges" >
          <property name="text" >
           <string>Draw boundary edges</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="clipPlaneGroup" >
     <property name="title" >
      <string>Clip plane</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="labelClipPX" >
          <property name="text" >
           <string>Point X:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="clipPointX" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text" >
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="labelClipNX" >
          <property name="text" >
           <string>Normal X:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QLineEdit" name="clipNormalX" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text" >
           <string>1.0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="labelClipPY" >
          <property name="text" >
           <string>Point Y:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="clipPointY" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text" >
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="labelClipNY" >
          <property name="text" >
           <string>Normal Y:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QLineEdit" name="clipNormalY" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text" >
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="labelClipPZ" >
          <property name="text" >
           <string>Point Z:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="clipPointZ" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text" >
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QLabel" name="labelClipNZ" >
          <property name="text" >
           <string>Normal Z:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" >
         <widget class="QLineEdit" name="clipNormalZ" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text" >
           <string>0.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>362</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Close</string>
       </property>
       <property name="icon" >
        <iconset theme="dialog-close" />
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Ok</string>
       </property>
       <property name="icon" >
        <iconset theme="dialog-ok-apply" />
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ElmerGUI.qrc" />
 </resources>
 <connections/>
</ui>
